metadata-ingestion/src/datahub/emitter/mce_builder.py

```diff
@@ -176,6 +176,10 @@ def make_data_job_urn_with_flow(flow_urn: str, job_id: str) -> str:
     return f"urn:li:dataJob:({flow_urn},{job_id})"
 
 
+def make_data_process_instance_urn(dataProcessInstanceId: str) -> str:
```

Comment:
if we're moving away from static builder methods towards urn classes, we don't add this?